frb_codegen/src/generator/dart/mod.rs

```diff
@@ -196,6 +207,25 @@ fn get_dart_imports(ir_file: &IrFile) -> HashSet<&IrDartImport> {
         .collect()
 }
 
+fn generate_wasm_module(funcs: &[IrFunc], dart_wire_class_name: &str) -> String {
+    format!(
+        "class {cls} extends FlutterRustBridgeWireBase {{
+            Future<void> init;
+            {cls}(WasmModule module): init = promiseToFuture(module())
+                .then((_) => eval('window.wasm_bindgen = wasm_bindgen'));
```

Comment:
For this two lines: Maybe can instead create a base class or a utility function in frb_dart support lib? By doing so, code is easier to maintain, to lint, to be processed by IDE etc.